drracket-tool-lib/drracket/private/standalone-module-browser.rkt

```diff
@@ -582,7 +574,7 @@
                   [parent menu]
                   [label (choice->label-string choice)]
                   [callback (λ (item evt)
-                              (hash-set! selected choice (not (hash-ref selected choice)))
+                              (hash-update! selected choice (λ (v) (not v)))
```

Comment:
maybe just `not`?

Reply:
(I don't really like `hash-update` and `hash-update!`, for the same reason that I really like `for` loops over `map` and friends. Not a big deal.)

Reply:
Filed jackfirth/resyntax#419.